packages/extension-store-locator/README.md

```diff
@@ -0,0 +1,92 @@
+# Store Locator Extension
+
+A PWA Kit extension that adds store locator functionality to your application. This extension provides a store locator solution with features like:
+
+- Search stores by postal code / country code via SCAPI Shopper Stores API
+- Device geolocation support
+- Configurable search radius
+- Multi-country support
+
+## Installation
+
+```sh
+npm install @salesforce/extension-store-locator
+```
+
+## Peer Dependancies
+
+PWA-Kit Application Extensions are NPM packages at their most simplest form, and as such you can define
+what peer dependencies are required when using it. Because this application extension provides
+Chakra UI via a page and components, it requires that the some peer dependencies are installed.
+
+Depending on what features your application extensions provides it's recommended you include any third-party
+packages as peer dependencies so that your base application doesn't end up having multiple versions of a 
+given package. See package.json for the full list of peer dependencies.
+
+## Configurations
+
+The Store Locator extension is configured via the `mobify.app.extensions` property in the config files or `package.json` file.
+
+```json
+{
+  "mobify": {
+    "app": {
+      "extensions": [
+        [
+          "@salesforce/extension-store-locator",
+          {
```

Comment:
This is nice. I wonder if we should follow a pattern like `tsconfig.json` files where we have the configuration formatted with inline comments and default values?

This might be an easy way to document our config. 

This isn't an action, more like a thought.